File: URI-c#/1008.cs
using System;

class Program
{
    static void Main()
    {
        int num, hours;
        double value;
        num = Convert.ToInt32(Console.ReadLine());
        hours = Convert.ToInt32(Console.ReadLine());
        value = Convert.ToDouble(Console.ReadLine());

        Console.WriteLine("NUMBER = {0:D}\nSALARY = U$ {1:0.00}",num,(hours*value));

    }
}

File: URI-c/1008.c
#include <stdio.h>

int main(){
	int num,hours;
	double price;
	scanf("%d %d %lf",&num,&hours,&price);
	printf("NUMBER = %d\nSALARY = U$ %.2f\n",num,price*hours);
	return 0;
}